services/blockchain-indexer/shared/dataService/business/events.js

```diff
@@ -50,11 +51,23 @@ const getEventTopicsIndex = () => getTableInstance(
 	MYSQL_ENDPOINT,
 );
 
+const eventCache = CacheLRU('events', config.endpoints.cache);
```

Comment:
Is it possible to use an in-memory cache that is not dependent on Redis?

Reply:
Yes, `cacheLRU` is not dependent on Redis. Will remove.